<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transform</class>
 <widget class="QMainWindow" name="Transform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>1129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCL可视化操作</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>901</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string> 深度图</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>由点云创建深度图</string>
      </property>
     </widget>
     <widget class="QVTKWidget" name="widget_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>761</width>
        <height>531</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>630</y>
        <width>491</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>深度图设置</string>
      </property>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>35</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Noise_Level</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="NOTBox_6">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>70</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="MNBBox_6">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>40</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min_Range</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="NOTBox_7">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>100</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Border_Size</string>
       </property>
      </widget>
      <widget class="QPushButton" name="executeButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>180</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>深度图转换</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loadSceneBut">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>加载目标点云</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>保存点云</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>彩色图</string>
     </attribute>
     <widget class="QVTKWidget" name="widget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>761</width>
        <height>631</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Z字段渲染图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadSceneBut_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>750</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>加载目标点云</string>
      </property>
     </widget>
     <widget class="QPushButton" name="executeButton_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>760</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行转换</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>760</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>保存点云</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>高度图</string>
     </attribute>
     <widget class="QVTKWidget" name="widget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>811</width>
        <height>601</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>高度图设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="executeButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>730</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行转换</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadSceneBut_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>730</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>加载目标点云</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>730</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>保存点云</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>灰度图</string>
     </attribute>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>灰度图设置</string>
      </property>
     </widget>
     <widget class="QVTKWidget" name="widget_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>811</width>
        <height>601</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="executeButton_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>720</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行转换</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadSceneBut_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>720</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>加载目标点云</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>720</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>保存点云</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>亮度图</string>
     </attribute>
     <widget class="QVTKWidget" name="widget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>761</width>
        <height>631</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>亮度图设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadSceneBut_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>740</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>加载目标点云</string>
      </property>
     </widget>
     <widget class="QPushButton" name="executeButton_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>740</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>执行转换</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>740</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>保存点云</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
